tests/tests/test_mtls.py

```diff
@@ -259,14 +281,17 @@ def test_mtls_enterprise(self, setup_ent_mtls, algorithm):
 
     @MenderTesting.fast
     @pytest.mark.parametrize("algorithm", ["rsa"])
-    def test_mtls_enterprise_hsm(self, setup_ent_mtls, algorithm):
+    @pytest.mark.parametrize("hsm_implementation", ["engine", "provider"])
```

Comment:
👍

You are sure it's ok to have two parametrizations at the same time?

(And unrelated, but should the other one have EC in it?)

Reply:
Yes, two work just fine :)

And we could _add_ EC as a test :) but it was not tested before, so leaving it out of this one I think